src/main/java/org/broadinstitute/hellbender/utils/SmallBitSet.java

```diff
@@ -0,0 +1,137 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeAlleleCounts;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeIndexCalculator;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+/**
+ * This class is very much like the standard library BitSet class in java.util, but instead of using
+ * a long[] array it uses single (32-bit) ints, which are much faster.  This limits the capacity but
+ * for small sets it makes sense.
```

Comment:
Make the comment about this louder and clearer.

Reply:
done